client/src/components/organizations/OrganizationSwitchPopover.vue

```diff
@@ -56,7 +104,11 @@ interface ConnectedOrganization {
   handle: ConnectionHandle;
 }
 
+const userInfo: Ref<ClientInfo | null> = ref(null);
+const isTrialOrg = ref(false);
```

Comment:
This needs to be set after this declaration: you can get the current available device with `getCurrentAvailableDevice()` from `@/parsec` and check the result with `isTrialOrganizationDevice(device)` from `@/common/organization` if the result is not undefined.